src/ontology/uberon-edit.obo

```diff
@@ -224665,7 +224663,6 @@ xref: BFO:0000067
 id: continuous_with
 name: continuous with
 xref: RO:0002150
-is_symmetric: true
```

Comment:
From [ticket](https://github.com/obophenotype/uberon/issues/2453#issuecomment-1118435290):

> [Typedef]
> id: continuous_with
> name: continuous with
> xref: RO:0002150
> is_symmetric: true
>  This should be added to RO https://github.com/oborel/obo-relations/pull/610

Do not see instruction to delete this.

Reply:
I'm cleansing the logical axioms already in RO.